sphinx_gallery/tests/test_notebook.py

```diff
@@ -136,7 +137,7 @@ def test_headings():
     ------------------------
     Blank heading above.
 
-                
+
```

Comment:
@larsoner I think the white space here is intentional, see below 'White space above' ?